chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/TestParticipantNoramlizer.java

```diff
@@ -0,0 +1,68 @@
+package gov.healthit.chpl.upload.listing.normalizer;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.dao.AgeRangeDAO;
+import gov.healthit.chpl.dao.EducationTypeDAO;
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.domain.TestParticipant;
+import gov.healthit.chpl.domain.TestTask;
+import gov.healthit.chpl.dto.AgeRangeDTO;
+import gov.healthit.chpl.dto.EducationTypeDTO;
+
+@Component
+public class TestParticipantNoramlizer {
+    private AgeRangeDAO ageRangeDao;
+    private EducationTypeDAO educationTypeDao;
+
+    @Autowired
+    public TestParticipantNoramlizer(AgeRangeDAO ageRangeDao, EducationTypeDAO educationTypeDao) {
+        this.ageRangeDao = ageRangeDao;
+        this.educationTypeDao = educationTypeDao;
+    }
+
+    public void normalize(CertifiedProductSearchDetails listing) {
+        if (listing.getSed() != null && listing.getSed().getTestTasks() != null
+                && listing.getSed().getTestTasks().size() > 0) {
+            listing.getSed().getTestTasks().stream()
+                .forEach(testTask -> {
+                    lookupTestParticipantAges(testTask);
+                    lookupTestParticipantEducationTypes(testTask);
```

Comment:
I think the naming needs changed here - based on the name I think these functions would return a looked up value.  I think they are actually looking up a value and setting the value in the object.  I think the name of the function should indicate what the side-effect of the function is.